### DISCUSSION TITLE
useExhaustiveDependencies - more flexibility in handling of stable functions

### DISCUSSION THREAD

**Discussion by ucwillg** [biomejs/biome#335] [Ideas]
useExhaustiveDependencies will apparently error if a hook includes a stable function in the dependency array, e.g.

```
const [value, setValue = useState(null);

const callback = useCallback(() => setValue(1), [setValue])
```

while the setValue isn't strictly needed since it should be stable, it is a matter of individual repo code style whether it should be included or not.

it would be helpful to be able to control the list of stable hooks or simply allow stable function to be included in the dependency array.

**Comment by ematipico** [comment]
How does the eslint plugin behave in this manner? 

Also, you can already pass a list of stable hooks to the rule: https://biomejs.dev/linter/rules/use-exhaustive-dependencies/#options
